src/sparseml/exporters/transforms/convert_quantizable_conv_integer.py

```diff
@@ -113,10 +54,13 @@ def convert_conv_to_quantized(
     :param input_quantize_node: The quantize node for the input of the conv node
     :return: The converted ONNX model
     """
-    graph = ONNXGraph(model)
+    input_quantize_node, input_dequantize_node = match.parents[0]
```

Comment:
I am just worried whether the numbering of the nodes is commutative.
I.e what such a situation be possible
A = match.parents[0]
B = match.parents[1]

as well  as

A = match.parents[1]
B = match.parents[0]

Reply:
the parents that are specified in the match definition are matched in the same order as the `node.input` values in the NodeProto. since we know this is a conv node, we know what each parent is.

i agree this is incredibly subtle. any idea how we could make it less subtle/more clear?

Reply:
this is subtle, but eventually, this is good news.
i think good docstrings will suffice, I would make them overly verbose if necessary